src/internal/modules/gauging-stations/forms/remove-tags-conditions.js

```diff
@@ -0,0 +1,68 @@
+const Joi = require('joi');
+
+const { formFactory, fields } = require('shared/lib/forms/');
+const session = require('../lib/session');
+const { groupLicenceConditions, addCheckboxFields } = require('../lib/helpers');
+const { createSchema } = require('shared/lib/joi.helpers');
+
+const removeTagsConditionsForm = request => {
+  const f = formFactory(request.path);
+  const mySession = session.get(request);
+  const dataLicenceConditions = groupLicenceConditions(request);
+  const dataUniqueLicencesLinkedTags = dataLicenceConditions.map(item => ({
+    licenceGaugingStationId: item.licenceGaugingStationId,
+    licenceId: item.licenceId,
+    licenceRef: item.licenceRef,
+    alertType: item.alertType,
+    thresholdValue: item.thresholdValue,
+    thresholdUnit: item.thresholdUnit,
+    dupeNum: item.linkages ? 1 : item.linkages.length,
+    linkages: addCheckboxFields(item.linkages)
+  })
+  );
+  const oneAndOnlyLicence = 0;
+  let selectedTags = [];
+  if (mySession.selectedLicence && dataUniqueLicencesLinkedTags) {
+    selectedTags = dataUniqueLicencesLinkedTags.filter(itemLabel => itemLabel.licenceId === mySession.selectedLicence.value);
+  }
+  /* Handle case when selected item already deleted */
+  const filteredChoices = !selectedTags.length ? [] : selectedTags[oneAndOnlyLicence].linkages;
```

Comment:
Might as well be `const filteredChoices = !selectedTags.length ? [] : selectedTags[0].linkages;` since `oneAndOnlyLicence` is not used anywhere else.